src/main/java/com/fabriccommunity/thehallow/mixin/ServerPlayNetworkHandlerMixin.java

```diff
@@ -101,14 +96,14 @@ public void onChatMessage(ChatMessageC2SPacket packet, CallbackInfo info) {
 	}
 	
 	private boolean isPlayerWearingCostume(PlayerEntity player) {
-		for (int i = 0; i < 4; i++) {
+		/*for (int i = 0; i < 4; i++) {
 			if (HallowedTags.Items.COSTUMES.contains(player.inventory.armor.get(i).getItem()))
```

Comment:
We don't need to comment out the tag check, do we? That can probably stay in for now.